entrypoint.sh

```diff
@@ -14,10 +18,12 @@ fi
 # --- Determine docker host address ------------------------------------------
 
 function _resolve_host {
-  getent ahostsv4 "$1" | head -n1 | cut -d' ' -f1
+  ip=$(getent ahostsv4 "$1" | head -n1 | cut -d' ' -f1)
```

Comment:
why the detour with an variable?

Reply:
The "unofficial strict mode" of bash makes the script exit as soon as any command exits with a non-zero status. For hosts that cannot be resolved (e.g. `host.docker.internal` will not resolve for podman users), this exits with a return value of 2, stopping further execution of the entrypoint script. I put the output of that `getent` command in a variable `ip`, so it doesn't stop further execution, plus it makes it clear what the output is. But if you want to turn it back into 1 line, I could change it to this: `echo $(getent ahostsv4 "$1" | head -n1 | cut -d' ' -f1)` instead?

Reply:
ok, as an alternative you could also just add `... || true` to the command pipe to prevent failing.